index.html

```diff
@@ -0,0 +1,132 @@
+          <p>
+            The dominance of mobile internet use means that users are searching
+            for the right business as they travel, shop, or sit on their couch
+            at home. Search Engine Optimization (SEO) allows you to increase
+            your visibility and find the right customers for your business.
+          </p>
+        </section>
+
+        <section id="online-reputation-management" class="paragraph-heading">
+          <img
+            src="./assets/images/online-reputation-management.jpg"
+            alt="A figure looks at his phone whilst they try to get company reputation up on a laptop"
+            class="float-right"
+          />
+          <h2>Online Reputation Management</h2>
+          <p>
+            The web is full of opinions, and some of these can be negative.
+            Social media allows anyone with an internet connection to say
+            whatever they want about your business. Online Reputation Management
+            gives you the control over what potential customers see when they
+            search for your business.
+          </p>
+        </section>
+
+        <section id="social-media-marketing" class="paragraph-heading">
+          <img
+            src="./assets/images/social-media-marketing.jpg"
+            alt="A group at a table discussing ideas"
+            class="float-left"
+          />
+          <h2>Social Media Marketing</h2>
+          <p>
+            Social media continues to have a sizable influence on buying habits.
+            Social media marketing helps you determine which platforms are
+            suited to your brand, using analytics to find the right markets and
+            increase your lead generation.
+          </p>
+        </section>
+      </div>
+
+      <section class="benefits">
+        <div class="benefit-1">
+          <h3>Lead Generation</h3>
+          <img
+            src="./assets/images/lead-generation.png"
+            alt="Looking for ideas = money"
+          />
+          <p>
+            Inbound strategies for lead generation require less work for your
+            business, bringing customers directly to your website.
+          </p>
+        </div>
+        <div class="benefit-1">
+          <h3>Brand Awareness</h3>
+          <img src="./assets/images/brand-awareness.png" alt="bulb" />
+          <p>
+            Users find your business through paid and organic searches,
+            increasing the search ranking and visibility for your business.
+          </p>
+        </div>
+        <div class="benefit-1">
+          <h3>Cost Management</h3>
+          <img
+            src="./assets/images/cost-management.png"
+            alt="Cog with dollar logo"
+          />
+          <p>
+            As the search ranking for your business increases, your advertising
+            costs decrease, and you no longer need to advertise your page.
+          </p>
+        </div>
+      </section>
+    </main>
```

Comment:
The `main` tag should be closed before the benefits section as the benefits are not part of the main content.

Reply:
I have made the change here.